_emails/northampton.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Northampton, MA
+permalink: "/northampton"
+name: Letter to City Council
+state: MA
+city: Northampton
+layout: email
+recipients:
+- glsciarra@northamptonma.gov
+- bdwight@comcast.net
+- mquinlanjr@northamptonma.gov
+- kfoster@northamptonma.gov
+- jimnashcitycouncil@gmail.com
+- jthorpe@northamptonma.gov
+- ajarrett@northamptonma.gov
+- mlabargeward6@aol.com
+- rmaiore@northamptonma.gov
+- lkrutzler@northmaioamptonma.gov
+subject: Reject the mayor's FY 2021 budget
+body: |-
+  Dear Councilor, 
+
+  My name is (NAME) and I live in the city of Northampton, MA. I’m writing to ask that City Council reject Mayor David Narkewicz’s proposal to expand police funding of nearly $200,000. In the budget, this proposed expansion is justified as necessary to mitigate the COVID-19 crisis and enforce “compliance with social distancing” (Narkewicz 2020). This despite the fact that experts across the country have urged against utilizing policing as a mechanism to respond to social and health problems (Gangi 2020). 
```

Comment:
Please use `[NAME]` instead of `(NAME)`. Brackets are less often used and not as easy to skim past

Reply:
For example you're also citing with `()`.

Reply:
updated